docs/gui/journal.rst

```diff
@@ -0,0 +1,42 @@
+gui/journal
+===========
+
+.. dfhack-tool::
+    :summary: Fort journal with a multi-line rich text editor
```

Comment:
I'd avoid calling this a rich text editor just yet. It doesn't support colors.

Reply:
updated